arkane/molpro.py

```diff
@@ -278,17 +278,21 @@ def loadEnergy(self, frequencyScaleFactor=1.):
         which it will parse out of the Molpro file. For the vdz and vtz basis sets f12a is
         a better approximation, but for higher basis sets f12b is a better approximation.
         """
-        E0 = None
+        e0 = None
         with open(self.path, 'r') as f:
             lines = f.readlines()
-            # Determine whether this is f12a or f12b according to the basis set, or whether this is MRCI.
-            f12a, f12b, mrci = False, False, False
+            # Determine whether the sp method is f12,
+            # if so whether we should parse f12a or f12b according to the basis set.
+            # Otherwise, check whether the sp method is MRCI.
+            f12, f12a, f12b, mrci = False, False, False, False
```

Comment:
Why do you need the `f12` variable? Could you just check that both `f12a` and `f12b` are false?

Reply:
We determine f12a and f12b from the basis set, so one of them could be True, but the method won't necessarily be F12

Reply:
They don't have to be on the same line if one uses an F12 method but not an F12 basis set (for which we have model chemistries)